### PR TITLE
TCM-442 | Setup firebase remote config connection

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] Setup the firebase remote config connection

### Issue

[TCM-442](https://hyfnla.atlassian.net/browse/TCM-442)

#### Firebase parameter added
Screen Shot 2021-06-17 at 6 02 12 PM

### CODE REVIEW

**Review by onodude — APPROVED** [review]
thanks!

File: app/src/main/java/com/trx/consumer/core/MainModule.kt [hunk 1]
```diff
@@ -133,7 +133,7 @@ object MainModule {
 
     @Provides
     @Singleton
-    fun provideConfigManager(): ConfigManager = ConfigManager()
+    fun provideConfigManager(cacheManager: CacheManager): ConfigManager = ConfigManager(cacheManager)
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/core/MainModule.kt:136]: Exceeds character limit

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Just one minor formatting issue

File: app/src/main/java/com/trx/consumer/core/MainModule.kt (hunk 1 above)
Comment (onodude) [reply app/src/main/java/com/trx/consumer/core/MainModule.kt:136]: should we add a return statement?

Comment (VivekV95) [reply app/src/main/java/com/trx/consumer/core/MainModule.kt:136]: Stylistically I prefer using `return` statements but looks like the pattern in the file seems to be just using `=`

Comment (devender197) [reply app/src/main/java/com/trx/consumer/core/MainModule.kt]: [Done](https://github.com/hyfn/trx-consumer-android/pull/121/commits/cf957b858f06569c9f92f87541c565fd216924f2)

**Review by onodude — APPROVED** [review]
thanks!

**Review by VivekV95 — APPROVED** [review]

**Pull request merged**